src/win/winsock.c

```diff
@@ -559,3 +559,34 @@ int WSAAPI uv_msafd_poll(SOCKET socket, AFD_POLL_INFO* info_in,
     return SOCKET_ERROR;
   }
 }
+
+int uv__convert_to_localhost_if_unspecified(const struct sockaddr* addr,
+	                                          struct sockaddr_storage* storage) {
+  if (addr == NULL) {
+    return UV_EINVAL;
+  } else if (addr->sa_family == AF_INET) {
+    const struct sockaddr_in* src;
+    struct sockaddr_in* dest;
+    src = (const struct sockaddr_in*) addr;
+    dest = (struct sockaddr_in*) storage;
+	  memcpy(dest, src, sizeof(struct sockaddr_in));
+    if (src->sin_addr.S_un.S_addr == 0) {
```

Comment:
incorrect capitalization?

Reply:
What do you mean?

Reply:
`src->sin_addr.s_addr` intead of the union.